src/components/buttons/СategoryButton/CategoryButton.constans.tsx

```diff
@@ -0,0 +1,18 @@
+import Room from '@assets/icons/category/room.svg';
+import Vegetable from '@assets/icons/category/vegetable.svg';
+import Fruit from '@assets/icons/category/fruit.svg';
+import Garden from '@assets/icons/category/garden.svg';
+
+export enum Categorys {
+	room = 'rooms',
+	garden = 'garden',
+	vegetable = 'vegetables',
+	fruit = 'fruits',
+}
```

Comment:
IMHO, these kinds of enums should come from backend

Reply:
Are we going to have only 4 options of categories? 
Is there an option that we will have more than 4 categories on future?

Reply:
Right now we're only looking at four categories, So it doesn't make sense to pull categories from the backend.

Reply:
Agree. Use python enum on backend to define data structure